src/utilities/test-utils.test.ts

```diff
@@ -2,6 +2,27 @@ import { TestUtils } from "./test-utils";
 import faker from "faker";
 
 describe("TestUtils", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region randomCase
+    // -----------------------------------------------------------------------------------------
+
+    describe("randomCase", () => {
+        test("it randomizes case of string", () => {
+            // Arrange & Act
+            let fixture = "";
```

Comment:
Minor: Could this be reduced to `const fixture = "a".repeat(20);`?

Reply:
Nice, thank you!